rfcs/63-nursery.md

```diff
@@ -0,0 +1,170 @@
+before eventually being merged in completely.
+When this occurs, nursery crate maintainers will be granted scoped merge rights to the main engine.
+
+## Motivation
+
+Third-party crates, commonly plugins, are often used to extend the engine in ways that are broadly applicable.
+Polished crates that are a good fit for the engine should eventually be merged in wholesale,
+rather than reimplemented, to avoid the [second system effect](https://en.wikipedia.org/wiki/Second-system_effect)
+and minimize breakage for their users.
+
+However, merging in these crates directly will result in very large, unreviewable PRs,
+and may compromise Bevy standards for quality.
+But if we delay, critical ecosystem crates may go stale, pointlessly splitting efforts, or making it hard for beginners to discover the de-facto standards.
+
+## User-facing explanation
+
+In addition to the core crates included within `bevy` itself, there are a number of **nursery** crates,
+which are being prepared for inclusion in the engine.
+
+These crates follow exactly the same technical rules as any other third-party crate, but:
+
+1. Live in the [`bevyengine` organization](https://github.com/bevyengine) on GitHub.
+2. Follow Bevy's standards for code quality and project management.
+3. Explicitly welcome community contributions to help get them ready for release.
+4. Will have a branch prepared for the newest release of Bevy before we release a new major version.
+
+You can find these crates by looking for the special `Nursery` tag on [Bevy Assets](https://bevyengine.org/assets/),
+or by browsing the [`bevyengine` organization](https://github.com/bevyengine)'s list of repositories.
+
+### Submitting a crate to the nursery
+
+If you would like to submit a crate for inclusion in the nursery, please open an issue in the `bevy-nursery` repository.
+While you do not need to be the crate's maintainer to do so, the following criteria must be met before it will be accepted to the nursery:
+
+1. The existing code must be MIT + Apache 2 licensed.
+2. The crate maintainers must consent.
+   1. While this is not strictly necessary due to licenses, buy-in and consent is important to us.
+3. The Bevy maintainers must collectively agree to accept the crate as a nursery crate.
+
+A crate is a good fit for the nursery if:
+
+1. The crate covers functionality that we eventually want to include in the engine itself. Factors include:
+   1. Broadly useful to engine users.
+   2. A direct replacement or improvement for an existing engine feature.
+2. The crate is reasonably mature.
+3. The crate is well-loved.
+4. The crate follows a high standard for testing, code quality and documentation.
+
+Very small or uncontroversial crates may be suitable for a direct PR via the ordinary mechanism.
+
+### Setting up a nursery crate
+
+Once accepted, the repository will be transferred to the Bevy organization under the same name and `crates.io` identity.
```

Comment:
In many cases the name may change, and it may also be a chance to make breaking changes that would otherwise be painful with a direct crate update.

For example, the name `bevy_mod_picking` was chosen to not conflict with the now reserved `bevy_picking`. Alternatively, it may even result in a merge with an existing crate (such as `bevy_ui`) instead of a simple crate addition to the monorepo.

Reply:
Agreed. It would be an excellent chance to evaluate whether it makes sense to change the repo name and/or the crate name (as published to crates.io).